packages/http-client-python/generator/pygen/codegen/templates/packaging_templates/README.md.jinja2

```diff
@@ -104,4 +104,35 @@ additional questions or comments.
 [pip]: https://pypi.org/project/pip/
 [azure_sub]: https://azure.microsoft.com/free/
 {% endif %}
+{% else %}
+# Overview
+
+This package is generated by `@typespec/http-client-python` with Typespec.
```

Comment:
Is it better to just talk about the package, not mention the code generator?

Reply:
@tadelesh  For unbranded SDK, users don't need set license header/"package-pprint-name"/package-name/namespace, so I really don't have idea how to describe the package-_- ...
Any thoughts?